style.css

```diff
@@ -0,0 +1,83 @@
+
+body {
+  font-family: 'Popins', sans-serif;
+  margin: 0;
+  min-height: 100vh;
+}
+
+a {
+  text-decoration: none !important;
+}
+
+header {
+  height: 10%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 0;
+}
+
+.mylogo {
+  width: 76px;
+  height: 20px;
+  margin-left: 8px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  padding-top: 58px;
+  padding-left: 24px;
+}
+
+.hum-menu {
+  padding-top: 68px;
+  padding-right: 36px;
+}
```

Comment:
- [OPTIONAL] Great job implementing the changes by the previous reviewer 💯 However, I suggest removing the `padding-top` property from `.mylogo` and `.hum-menu` classes or decrease it to match your Figma design 👍